src/pages/workspace/accounting/qbo/export/QuickbooksCompanyCardExpenseAccountSelectCardPage.tsx

```diff
@@ -49,19 +56,34 @@ function QuickbooksCompanyCardExpenseAccountSelectCardPage({policy}: WithPolicyC
                 value: CONST.QUICKBOOKS_NON_REIMBURSABLE_EXPORT_ACCOUNT_TYPE.VENDOR_BILL,
                 keyForList: CONST.QUICKBOOKS_NON_REIMBURSABLE_EXPORT_ACCOUNT_TYPE.VENDOR_BILL,
                 isSelected: CONST.QUICKBOOKS_NON_REIMBURSABLE_EXPORT_ACCOUNT_TYPE.VENDOR_BILL === nonReimbursableExpensesExportDestination,
+                accounts: accountPayable ?? [],
+                defaultVendor: vendors?.[0]?.id ?? CONST.INTEGRATION_ENTITY_MAP_TYPES.NONE,
```

Comment:
The default vendor [should be NONE by default according to this part of the design doc](https://docs.google.com/document/d/1LLrOYRGEb_k_Z6dttcS99zlbCNNdpD1QlXAbFJYbR7g/edit#bookmark=id.o8ubp9u8fe50)

<img width="400" alt="Screenshot 2024-05-06 at 11 05 42 AM" src="https://github.com/Expensify/App/assets/98560306/5b183083-d572-4700-a4e5-0bffa20d3ccb">

Reply:
^^ that's referring to the toggle for default vendor being off by default when you connect to QBO.

Reply:
yes off -> `NONE` 😄

Reply:
Cool yes, the toggle should be off (and the `Vendor` push input not shown) until the user toggles `Default vendor` on. 👍

Reply:
I think you may be misunderstanding the code, this is the default value that will be used if the account type changes to VENDOR_BILL. I understand that in that case we should select the first vendor.